### PR TITLE
Apple silicon(M1) compatibility fix

### LINKED ISSUE #2: Apple silicon compatibility
Hi,
As I was setting up I have noticed that the repo is using an old version of the node which is not compatible with apple silicon(aka M1) architecture. Primarily due to usage of node-sass.
Luckily the fix is quite [simple](https://sass-lang.com/blog/libsass-is-deprecated)
I made a PR with the fix.

### PR DESCRIPTION
**Pull request by Maxim-Filimonov**
- update to node compatible with m1
- replace node-sass with sass.
Resolves #2

### CODE REVIEW

File: .tool-versions [hunk 1]
```diff
@@ -1,3 +1,3 @@
 elixir 1.11.2-otp-23
-erlang 23.1
-nodejs 12.14.1
+erlang 23.3.4.3
```
Comment (Maxim-Filimonov) [inline .tool-versions:2]: not sure why but erlang 23.1 doesn't install on m1 either. The version specified works and tested

**Review by Maxim-Filimonov — COMMENTED** [review]
See note about the erlang version, I can try to investigate the installation issue if it's critical to stay on that version. It's a minor upgrade so...

**Comment by germsvel** [comment]
@Maxim-Filimonov thank you so much for opening this PR. I'll go ahead and install all these versions and make sure everything is working as expected. If everything looks good, I'll merge the PR. 

I'll then have to go and update all the branches to use the new versions, and that might take me a little bit. But please, let me know if you run into any issues while using this in the meantime, and I'll do my best to help.

**Comment by germsvel** [comment]
@Maxim-Filimonov just wanted to loop back and let you know that all the branches have been updated to include the change introduced in this PR. Thanks for the patience!

**Comment by Maxim-Filimonov** [comment]
> @Maxim-Filimonov just wanted to loop back and let you know that all the branches have been updated to include the change introduced in this PR. Thanks for the patience!

Great 😊 Thank you for incorporating the changes.

**Pull request merged**